app/graphql/types/customer/customer.gql

```diff
@@ -27,8 +27,40 @@ type CustomerAuthenticationQuestions {
   memorablePlace: String
 }
 
+type CustomerApplication {
```

Comment:
Lots of stuff here I don't think we need, please ask Ian Ball or James Keep about what data we need, but I would work off the RPA CCV requirements spreadsheet unless you find otherwise